bedrock/firefox/views.py

```diff
@@ -509,7 +509,11 @@ def new(request):
     locale = l10n_utils.get_locale(request)
 
     if scene == '2':
-        if locale == 'en-US':
+        if lang_file_is_active('firefox/new/batm', locale) and experience == 'batmprivate':
+            template = 'firefox/new/batm/scene2.html'
+        elif lang_file_is_active('firefox/new/sem', locale) and experience == 'fast':
```

Comment:
Until we have a better solution, maybe add a comment that the above are only intended to be `en-US` and `de`? (I think that's also the case for `batmprivate`...)